src/docs/development/add-to-app/android/add-flutter-fragment.md

```diff
@@ -359,10 +359,7 @@ val flutterFragment = FlutterFragment.withNewEngine()
 
 Similar to varying initial routes, different `FlutterFragments` may want to
 execute different Dart entrypoints. In a typical Flutter app, there is only one
-Dart entrypoint: `main()`, but you can define other entrypoints. See [defining
-Dart entrypoints] for instructions about defining such entrypoints.
-
-[defining Dart entrypoints]: /docs/development/platform-integration/add-to-app-android/custom-dart-entrypoints
+Dart entrypoint: `main()`, but you can define other entrypoints.
```

Comment:
TBR @matthew-carroll

There's a broken link here and the linked page was never written. I removed it for now.